### ISSUE TITLE
Authentication window closes unexpectedly

### ISSUE THREAD

**Issue by Skirmisher** [eul-im/eul#40]
The authentication browser window closes unexpectedly every time I attempt to use it. Usually it will last for several seconds, but it never stays open for more than a minute (perhaps not even 30 seconds). This renders any attempt to authenticate unsuccessful, as the browser closes before a login can complete (especially one involving 2FA).

Each failure leaves this message in the browser debug.log:
```
[0920/123943.355:FATAL:message_window.cc(67)] Check failed: result. 
```

This seems to be unrelated to the particular login page being displayed, nor any actions taken on it; I have observed the issue with the Skype and Slack login pages (the latter I took no actions on).

This is rather unfortunate, as it renders eul useless. :(

**Comment by voltapp2** [comment]
Hi @Skirmisher 

The browser is closed automatically after 40 seconds. I will change it to 2 minutes. That should be enough for everyone.

This will be live once v0.26 is released. This is a pretty big release, but I'm doing my best to make it go live by tomorrow.

**Comment by voltapp2** [comment]
Fixed in v0.26
